jsLearn/arrayMethods.js

```diff
@@ -0,0 +1,181 @@
+// 1. Translate border-left-width to borderLeftWidth
+
+function camelize(str) {
+    const word = str.split('-').map((item, index) => index === 0 ? item : item[0].toUpperCase() + item.slice(1)).join('');
+
+    return word
+}
+
+let string1 = 'abc-defgh-igklmn';
```

Comment:
build a behaviour to use const if you'll not gonna change the value of the variable
```suggestion
const string1 = 'abc-defgh-igklmn';
```

Reply:
ok, done